README.md

```diff
@@ -144,7 +144,7 @@ Grid.rectangle({ width: 100, height: 100 }).forEach(hex => {
 #### With [SVG.js](http://svgjs.com/)
 
 ```javascript
-const draw = SVG(document.body)
+const draw = SVG().addTo('body').size(300, 300)
```

Comment:
I think it would be a little better like so:

```suggestion
const draw = SVG().addTo('body').size('100%', '100%')
```

I've updated the jsfiddle I'm referencing in the docs the same way.